modules/dnn/perf/perf_convolution.cpp

```diff
@@ -878,17 +882,106 @@ PERF_TEST_P_(Conv, conv)
         << "    Weights(parameters): " << divUp(weightsMemory, 1u<<10) << " Kb"
         << "    MFLOPS=" << flops * 1e-6 << std::endl;
 
+    EXPECT_NEAR(flops, declared_flops, declared_flops * 1e-6);
+
+    return net;
+}
+
+typedef tuple<ConvParam_t, bool, tuple<Backend, Target> > ConvTestParam_t;
+typedef TestBaseWithParam<ConvTestParam_t> Conv;
+typedef TestBaseWithParam<ConvTestParam_t> Conv_1x1;
+typedef TestBaseWithParam<ConvTestParam_t> Conv_3x3S1D1;
+typedef TestBaseWithParam<ConvTestParam_t> Conv_Depthwise;
+
+PERF_TEST_P_(Conv, conv)
+{
+    const ConvParam_t& params = get<0>(GetParam());
+    Backend backendId = get<0>(get<2>(GetParam()));
+    Target targetId = get<1>(get<2>(GetParam()));
+    bool winograd = get<1>(GetParam());
+    Net net = build_net(params, backendId, targetId);
+    net.enableWinograd(winograd);
```

Comment:
There is "warmup" stage in the original test code.

If we change settings, then we should do that again.

Reply:
warm up happen on `build_net()` function. I didn't change this part of the code

Reply:
If you play with network configuration setting (like `net.enableWinograd(winograd);` on the line 932) then you should do "warmup" again.

Reply:
Thanks! It's right. I will do it soon